lib/swipe_cards.dart

```diff
@@ -135,7 +137,11 @@ class _SwipeCardsState extends State<SwipeCards> {
       case Decision.like:
         return SlideDirection.right;
       case Decision.superLike:
-        return SlideDirection.up;
+        if (widget.draggableTop == true) {
+          return SlideDirection.up;
+        } else {
+          return null;
+        }
```

Comment:
Klo `draggableTop` ga pake `?` harusnya ini jdi simply
```suggestion
        if (widget.draggableTop) {
          return SlideDirection.up;
        } 
        return null;
```